TelCo.ColorCoder/Utils/ColorCodeConstant.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Text;
+
+namespace TelCo.ColorCoder.Utils
+{
+    public static class ColorCodeConstant
```

Comment:
these are not necessarily constant, remove constants from class name. move the class to TelCo.ColorCoder namespace. You can call the class ColorCodeRegistry for better representation

Reply:
Done.